plugins/tiddlywiki/multiwikiserver/modules/routes/handlers/delete-acl.js

```diff
@@ -0,0 +1,41 @@
+/*\
+title: $:/plugins/tiddlywiki/multiwikiserver/routes/handlers/delete-acl.js
+type: application/javascript
+module-type: mws-route
+
+POST /admin/delete-acl
+
+\*/
+(function () {
+
+	/*jslint node: true, browser: true */
+	/*global $tw: false */
+	"use strict";
+
+	var aclMiddleware = require("$:/plugins/tiddlywiki/multiwikiserver/modules/routes/helpers/acl-middleware.js").middleware;
+
+	exports.method = "POST";
+
+	exports.path = /^\/admin\/delete-acl\/?$/;
```

Comment:
Same as above: `/api/v1/alc/<resource>` -- I am not sure if the path needs to contain the action `delete` since it is defined by the HTML command DELETE -- IMO we only need to specify the resource that should be deleted.